src/test/java/org/mockitousage/stubbing/BuilderStubbingTest.java

```diff
@@ -0,0 +1,98 @@
+package org.mockitousage.stubbing;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.RETURNS_SELF;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class BuilderStubbingTest {
```

Comment:
Also as this class is about usage, I'd try to make a whole example of code for a builder, for a builder that behaves like this for example.

```java
Built built = someBuilder.withA(...)
                         .withB(...)
                         .withC(...)
                         .build()
```